terraform/create.go

```diff
@@ -163,6 +164,16 @@ func (t *Terraform) Create() error {
 }
 
 func (t *Terraform) preFlightCheck() error {
+	if os.Getenv("SSH_AUTH_SOCK") == "" {
+		return fmt.Errorf("ssh agent not running. Please run eval \"$(ssh-agent -s)\" and then ssh-add.")
+	}
+	if len(t.config.DeploymentConfiguration.DBPassword) <= 8 {
+		return fmt.Errorf("db password needs to be longer than 8 characters")
+	}
```

Comment:
I believe the password needs to be at least 8, so a length of 8 would be valid.

Reply:
Fixed.